README.md

```diff
@@ -66,7 +66,7 @@ See the [CONTRIBUTING.md] for more ways to contribute and instructions.
 
 Viskores Requires:
 
-  + C++14 Compiler. Viskores has been confirmed to work with the following
+  + C++17 Compiler. Viskores has been confirmed to work with the following
       + GCC 5.4+
       + Clang 5.0+
       + XCode 5.0+
```

Comment:
@vicentebolea Has this list been updated? This list should probably match the compilers used in the CI.

Reply:
Sounds good, this values are correct circa VTKm 2.3.0 (sans cuda, we need Pascal arch).